testsuite/classlibrary/TestReadableNodeIDAllocator.sc

```diff
@@ -0,0 +1,85 @@
+TestReadableNodeIDAllocator : UnitTest {
+
+	// test proper creation from Server
+	test_readableNodeRange {
+		var s = Server(\testRNIA);
+		var prevClass = Server.nodeAllocClass;
+		Server.nodeAllocClass = ReadableNodeIDAllocator;
+		s.options.maxLogins = 1; s.newAllocators;
+		// this.assert(
+		// 	(s.nodeAllocator.numIDs == (10 ** 8)),
+		// 	"for a single client, (readable) nodeAllocator has maximum range."
+		// );
+		// this.assert(
+		// 	(s.nodeAllocator.numIDs == (10 ** 7)),
+		// 	"for 16 clients, (readable) nodeAllocator has divided range."
+		// );
+
+		// same for ReadableNodeIDAllocator
+		// Server.nodeAllocClass = ReadableNodeIDAllocator;
+		Server.named.removeAt(s.name); Server.all.remove(s);
```

Comment:
reminder to put these on separate lines or move into a helper function within this class

Reply:
done -  added a Server:remove method, as it is generally useful be able to completely remove a server in a single clear method.